api/conftest.py

```diff
@@ -63,6 +67,51 @@
 )
 from users.models import FFAdminUser, UserPermissionGroup
 
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--ci",
+        action="store_true",
+        default=False,
+        help="Enable CI mode",
+    )
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config: pytest.Config) -> None:
+    if (
+        config.option.ci
+        and config.option.dist != "no"
+        and not hasattr(config, "workerinput")
```

Comment:
It seems weird that if `ci` is present that if `workerinput` is set that this will silently fail. Don't we prefer an exception if `ci` is set and so is `workerinput`?

Reply:
Having `ci` and having `workerinput` is a legitimate case when we need this hook to be a noop.